index.html

```diff
@@ -192,12 +192,12 @@ <h2>How do you determine: Fast, Average, Slow?</h2>
 
           <ul>
             <li>
-              “fast” TTFB &lt; 200 ms is based on the <a href="https://developers.google.com/speed/docs/insights/Server">server responsiveness</a>
+              “fast” TTFB &lt; 500 ms is based on the <a href="https://developers.google.com/speed/docs/insights/Server">server responsiveness</a>
```

Comment:
@rviscomi missed this part :)
The link is actually deprecated, I'm not sure it's still relevant, but didn't have something better to link to.. got something better?